cmd/argocd-server/commands/root.go

```diff
@@ -49,7 +49,7 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringVar(&staticAssetsDir, "staticassets", "", "Static assets directory path")
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().StringVar(&repoServerAddress, "repo-server", "localhost:8081", "Repo server address.")
-	command.Flags().StringVar(&configMapName, "config-map", "", "Name of a Kubernetes config map to use.")
+	command.Flags().StringVar(&configMapName, "config-map", "argo-cd-cm", "Name of a Kubernetes config map to use.")
```

Comment:
Might it make sense to replace `"argo-cd-cm"` with a `util.ConfigManagerDefaultConfigMapName` const?  That way the same default const can be used between `argocd install` and `argocd-server`.

Reply:
@merenbach for some reason I cannot find ConfigManagerDefaultConfigMapName const. Is it in master or some other branch?

Reply:
@alexmt I should have clarified.  This would be a new const in `util/configmanager.go`.

Reply:
Got it. It make sense, working on changes